src/main/java/org/openpnp/machine/reference/vision/ReferenceBottomVision.java

```diff
@@ -87,7 +87,6 @@ private static PartAlignmentOffset findOffsetsPreRotate(Part part, BoardLocation
                            .getRotation();
         }
         angle = angleNorm(angle, 180.);
-        double placementAngle = angle;
```

Comment:
We take the angle from the `location` instead. Works smoother with the multi-pass vision loop, where the `location` is gradually adapted.